compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1667,6 +1667,164 @@ static LogicalResult setElementwiseGenericOpRootConfig(
                                                tileSizes, passPipeline);
 }
 
+// Checks if the passed op is a dequantization on grouped input
+// This function checks that the genericOp:
+// 1. Has a body like:
+//      arith.extui
+//      arith.uitofp
+//      arith.subf
+//      arith.mulf
+//      arith.mulf
+//      arith.addf
+// 2. Increases the bit width of the input
+// 3. Has 3 parallel dims
+// 4. Has 4 (rhs, weights, scales, zero points)
```

Comment:
Wondering if it make sense to introduce a higher level op to represent this... Matching all these specific details out of a generic of seems error prone

Reply:
Ah yeah, the more we try to match these quantization generics, the more it seems to make sense to do that. The changes to KernelDispatch.cpp shouldn't really be in this PR though. I was just using that to play with tile sizes for now. The main important changes here in this PR are in `FuseDequantizationMatmul.cpp`. I can close the PR for now and leave it as a branch until we have something more concrete if that is better